source/developer/localization.rst

```diff
@@ -103,33 +103,33 @@ If you're interested in contributing to the process, please join the `Mattermost
 +------------------------------------------+-----------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------+
 | Language                                 | Official Reviewer(s)                                                                                                  | Maintainers                                                                                          |
 +==========================================+=======================================================================================================================+======================================================================================================+
-| Deutsch - German                         | `Christian Arnold (meilon) <https://github.com/meilon>`__                                                              | `Tim Estermann (der-test) <https://github.com/der-test>`__                                            |
```

Comment:
Some of the linting fixes caused the tables to break, e.g. below. Curious why that might be?

![image](https://user-images.githubusercontent.com/13119842/50655047-e1ba4680-0f5c-11e9-939f-0e150c319d68.png)

Reply:
Looks like a) I missed that, and b) it curiously didn't generate a warning itself. I've pushed up a change to address this, and scanned the other tables in a bid to convince myself I didn't make the same mistake elsewhere.

It's possible something else in the warning output was complaining about this, but it's still obscured by the other warnings.